components/event-sources/cmd/sources-controller/main.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2019 The Kyma Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	// allow client authentication against GKE clusters
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	"knative.dev/pkg/injection/sharedmain"
+
+	"github.com/kyma-project/kyma/components/event-sources/reconciler/mqttsource"
+)
+
+func main() {
+	sharedmain.Main("sources_controller",
```

Comment:
I was actually wondering why, because this manager manages multiple Source**s**. Are you absolutely tied to the plural name?

Reply:
I think because the component name is used for the logger name of each controller inside a manager. I am not tied to the singular name. Choose whichever name you prefer.